spec/domain/healthchecks/etl_google_analytics_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe Healthchecks::EtlGoogleAnalytics do
+  its(:name) { is_expected.to eq(:etl_google_analytics_pviews) }
+
+  include_examples 'Healthcheck enabled/disabled within time range'
+
+  subject { described_class.build(:pviews) }
+
+  let(:yesterday) { Date.yesterday }
+
+  context 'when there are no pviews' do
+    before do
+      edition = create :edition
+      create :metric, edition: edition, date: yesterday, pviews: 0
+      create :metric, edition: edition, date: Date.today, pviews: 10
+    end
+
+    its(:status) { is_expected.to eq(:critical) }
+    its(:message) { is_expected.to eq('ETL :: no pviews for yesterday') }
+  end
+
+  context 'when there are no pviews for yesterday' do
```

Comment:
Not sure this context is correct? - Should this be `when there are pviews for yesterday`?